pkg/labels/labels.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package labels
+
+import (
+	"strings"
+)
+
+// Labels allows you to present labels independently from their storage.
+type Labels interface {
```

Comment:
why do you have this interface here?

Reply:
We couldn't add a Labels interface as there was conflict in the hvtag package

Reply:
But why do you need it at all? You cannot redefine it anway ifwe use Kubernetes libraries

Reply:
ahh sorry I completely went with the flow of copy pasting and forgot to remove the Labels interface, as our type should satisfy it and that's what we need